src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml

```diff
@@ -1,74 +1,98 @@
-                    </ViewCell>
-                </TableSection>
-                <TableSection Title="Define new condition">
-                    <ViewCell>
-                        <Grid>
-                            <Grid.RowDefinitions>
-                                <RowDefinition Height="auto" />
-                                <RowDefinition Height="auto" />
-                                <RowDefinition Height="auto" />
-                                <RowDefinition Height="auto" />
-                            </Grid.RowDefinitions>
-                            <Picker x:Name="Attributes"
-                                    Grid.Row="0"
-                                    ItemDisplayBinding="{Binding Name}"
-                                    SelectedIndexChanged="OnAttributeChanged"
-                                    VerticalOptions="CenterAndExpand" />
-                            <Picker x:Name="Operators"
-                                    Grid.Row="1"
-                                    VerticalOptions="CenterAndExpand" />
-                            <Picker x:Name="ValueSelection"
-                                    Grid.Row="2"
-                                    ItemDisplayBinding="{Binding Name}"
-                                    VerticalOptions="CenterAndExpand" />
-                            <Entry x:Name="ValueEntry"
-                                   Grid.Row="2"
-                                   IsVisible="False" />
-                            <Button Grid.Row="3"
-                                    Clicked="OnAddCondition"
-                                    Text="Add condition" />
-                        </Grid>
-                    </ViewCell>
-                </TableSection>
-                <TableSection Title="Barrier conditions">
-                    <ViewCell>
-                        <ScrollView HeightRequest="150">
-                            <Label x:Name="ConditionBarrierExpression"
-                                   Grid.Row="0"
-                                   FontAttributes="Bold"
-                                   Text="" />
-                        </ScrollView>
-                    </ViewCell>
-                </TableSection>
-                <TableSection>
-                    <ViewCell>
-                        <Button Clicked="OnTrace" Text="Trace" />
-                    </ViewCell>
-                    <ViewCell>
-                        <Button Clicked="OnReset" Text="Reset" />
-                    </ViewCell>
-                </TableSection>
-            </TableRoot>
-        </TableView>
-    </Grid>
+﻿<ContentPage
+    x:Class="ArcGISRuntime.Samples.ConfigureSubnetworkTrace.ConfigureSubnetworkTrace"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+    <ScrollView MinimumHeightRequest="1000">
```

Comment:
A maximum width might be useful here as the UI becomes very stretched in horizontal orientation on mobile.